tool/teleport/common/teleport.go

```diff
@@ -477,6 +483,14 @@ func onConfigDump(flags dumpFlags) error {
 		return trace.Wrap(err)
 	}
 
+	entries, err := os.ReadDir(flags.DataDir)
```

Comment:
I would at least log `err` unless it's not "does not exist".

Reply:
Agreed.

I added a new error check at commit https://github.com/gravitational/teleport/pull/11766/commits/723e5703ae21c9595ad5408341fca5a9994db2ca and log to stderr in case we could not check the directory. Let me know what you think